packages/web-react/src/components/TooltipModern/demo/TooltipWithFloatingUI.tsx

```diff
@@ -0,0 +1,144 @@
+          label="Enable flipping cross axis"
+          onChange={() => setFlipCrossAxis(!flipCrossAxis)}
+        />
+        <Checkbox id="shift" isChecked={shift} label="Enable shifting" onChange={() => setShift(!shift)} />
+        <Checkbox id="size" isChecked={size} label="Enable sizing" onChange={() => setSize(!size)} />
+      </Grid>
+      <Grid cols={{ mobile: 1, tablet: 2, desktop: 2 }} UNSAFE_className="mb-700">
+        <Select
+          id="advanced-placement"
+          name="advanced-placement"
+          label="Suggested placement"
+          ref={suggestedPlacementRef}
+          onChange={handleSuggestedPlacementChange}
+          defaultValue="top-start"
+        >
+          <option value="top-start">top-start</option>
+          <option value="top">top</option>
+          <option value="top-end">top-end</option>
+          <option value="right-start">right-start</option>
+          <option value="right">right</option>
+          <option value="right-end">right-end</option>
+          <option value="bottom-start">bottom-start</option>
+          <option value="bottom">bottom</option>
+          <option value="bottom-end">bottom-end</option>
+          <option value="left-start">left-start</option>
+          <option value="left">left</option>
+          <option value="left-end">left-end</option>
+        </Select>
+        <Select
+          id="advanced-placement-fallback"
+          name="advanced-placement-fallback"
+          label="Suggested fallback placement"
+          ref={suggestedFallbackPlacementRef}
+          onChange={handleSuggestedFallbackPlacementChange}
+          defaultValue="top, right, bottom"
+        >
+          <option value="top, right, bottom">top, right, bottom</option>
+          <option value="top, left, bottom">top, left, bottom</option>
+          <option value="top, left, right, bottom">top, left, right, bottom</option>
+          <option value="bottom, right, top">bottom, right, top</option>
+          <option value="top, bottom">top, bottom</option>
+          <option value="bottom, top">bottom, top</option>
+        </Select>
+      </Grid>
+      <div
+        className="bg-cover"
+        style={{ width: '100%', maxWidth: '100%', height: '30rem', overflow: 'auto' }}
+        ref={viewportRef}
+      >
+        <div
+          style={{ position: 'relative', width: '300%', height: '90rem', paddingBlock: '44rem', textAlign: 'center' }}
+          ref={contentRef}
+        >
+          <TooltipModern
+            id="TooltipModernAdvancedExample"
+            isOpen={isOpen}
+            onToggle={setIsOpen}
+            placement={suggestedPlacement}
+            enableFlipping={flip}
+            enableFlippingCrossAxis={flipCrossAxis}
+            flipFallbackPlacements={suggestedFallbackPlacement}
+            enableShifting={shift}
+            enableSizing={size}
+          >
+            <TooltipTrigger elementType={Button}>I have a flipping tooltip!</TooltipTrigger>
```

Comment:
Can we unify this text with other packages?

Reply:
texts unified